include/alpaka/kernel/TaskKernelGenericSycl.hpp

```diff
@@ -111,16 +110,6 @@ namespace alpaka::detail
     {
     };
 
-    template<typename TElem, typename TIdx, std::size_t TDim, typename TAccessModes>
-    inline auto require(
-        sycl::handler& cgh,
-        experimental::Accessor<SyclAccessor<TElem, DimInt<TDim>::value, TAccessModes>, TElem, TIdx, TDim, TAccessModes>
-            acc,
-        special)
-    {
-        cgh.require(acc.m_accessor);
-    }
```

Comment:
@j-stephan I don't quite understand what the replacement is here after the USM merge. Do we still need to require things anymore? And if yes, where do they come from?